src/write.rs

```diff
@@ -194,6 +194,46 @@ impl ZipWriterStats {
     }
 }
 
+impl<A: Read + Write + io::Seek> ZipWriter<A> {
+    /// Initializes the archive from an existing ZIP archive, making it ready for append.
+    pub fn new_append(mut readwriter: A) -> ZipResult<ZipWriter<A>> {
```

Comment:
I'd like to simply call this `open`, but I think we can bikeshed it for a bit before release

Reply:
Will keep this for an overall API change, since currently both the `ZipArchive` constructor and `ZipWriter<Write + Seek>` constructor have the name `new`.